core/domain/rights_manager.py

```diff
@@ -1137,7 +1057,13 @@ def _publicize_activity(committer_id, activity_id, activity_type):
     Raises:
         Exception. The committer does not have rights to publicize the activity.
     """
-    if not Actor(committer_id).can_publicize(activity_type, activity_id):
+    committer_role = user_services.get_user_role_from_id(committer_id)
+    committer_actions = role_services.get_all_actions(committer_role)
+    activity_rights = _get_activity_rights(activity_type, activity_id)
+
+    # Publicizing functionality is going to be deprecated. That's why check
+    # implemented only for exploration and not for both.
```

Comment:
do I still need to do this. (As publicizing is going to be deprecated before next release)